### ISSUE TITLE
windows todeClient malfunctioning

### ISSUE THREAD

**Issue by HeSe** [GsDevKit/gsDevKitHome#73]
Hi!

I just tried to install the tODE client on windows and followed the documented process.
Once the installation is done and I am faced with the pharo todeClient image there are no entries in the tODE Edit submenu.
Also clicking on todeClient update or tODE Shell does not show any effect

**Comment by dalehenrich** [comment]
@HeSe, I think the issue is that for remote clients, when you finish step 6, you also need to push your commit to your github account, since your gemstone server has the stone description (I asume that's what is missing) ... this is missing documentation and I'll need to fix it ... in the mythical, upcoming 1.0.0 release:)

**Comment by HeSe** [comment]
Hi Dale

There is no git account for my server. I cloned both client and server 
gsDevKit on different machines.

But this helped to further understand the process...

So I assume I need to modify the SessionSetup script in tODE menu
tODE - tODE Workspace - SeassionSetup.ws

So for windows that would then look like e.g.

| gsDevKitRoot sessionName stoneName |
gsDevKitRoot := 'C:\Users\xyyz\gsDevKitHome\'.
sessionName := 'gsDevKit'.
stoneName := 'gsDevKit'.
TDShell devKitRoot: gsDevKitRoot.
(TDSessionDescription new
     name: sessionName;
     gemstoneVersion: 'myGSVersion';
     stoneHost: 'localhost';
     stoneName: 'gsDevKit';
     gemHost: 'myIP';
     netLDI: 'myStonePort';
     gemTask: 'gemnetobject';
     userId: 'DataCurator';
     password: 'swordfish';
     osUserId: '';
     osPassword: '';
     backupDirectory: '';
     dataDirectory: '';
     serverGitRoot: '/opt/git';
     serverTodeRoot: gsDevKitRoot, '/dev/tode';
     yourself) exportTo: TDShell sessionDescriptionHome.
TDShell testLogin: sessionName.

Executing this creates an entry in my desired tODE entry!


This worked fine!
Well I wasn'T really sure about the content for gsDevKitRoot but it 
became clear it must be the client sided gsDevKitHome path...

Okay!
It works!

Thank you!
Sebastian

**Comment by HeSe** [comment]
Was ther ea change?

I do remeber that there was at least a default devKit entry once?!...

Sebastian

**Comment by dalehenrich** [comment]
Well, you do already have a fork of gsDevKitHome: 
https://github.com/HeSe/gsDevKitHome and that is the github project that 
you should be pulling and pushing from to and sharing the changes you 
make on your server with the your remote client and vice versa ...

@rjsargent has been working the the github windows client and I think 
that he is having good success with using it as it simplifies the 
interaction with git on your client (once you run through the initial 
install steps) ...

Dale

**Comment by dalehenrich** [comment]
When you run installServer on the gemstone linux box, there is a tode/client/description/<stone-name> file that gets created ... that file needs to be sent to your remote client (scp, git, rsynch, copy.paste) and that is the file (for each stone) that tells your client how to login to the gemstone server and it needs to be on the local disk on the client machine ...

Once you get logged in, most of the files and disk-based artifacts are resident on the gemstone linux server... pretty much the description files and a few bootstrap files are all that's needed on the client ...

**Comment by rjsargent** [comment]
One thing to bear in mind about replicating the descriptions to multiple machines.

The default description from the server install uses "localhost" as the host name. You need to edit that to the server's actual hostname if you plan to access it from another machine (such as a Windows client).

**Comment by dalehenrich** [comment]
Yeah, I explicitly mention that you need to change the netldi port number in the doc .. so I need to add a bit about the hostname/ipaddress
